src/Str.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace OL\String;
+
+use OL\String\Contracts\Str as StrContract;
+
+/**
+ * Class Str.
+ *
+ * Basic manipulation of strings.
+ */
+class Str implements StrContract
+{
+    /**
+     * @var string Raw string content
+     */
+    protected $payload;
```

Comment:
I'd rather have this be named `content` or something else. Personal preference, only. Payload reminds me of multitype data, blobs, text, etc. We're talking about text-only data.